src/plugin/initialize.sql

```diff
@@ -0,0 +1,241 @@
+CREATE TABLE IF NOT EXISTS crosswalk (
```

Comment:
In an ideal world, we break this file up into smaller portions and load those files instead of hard coding the code in this file.. in another file as well.